README.md

```diff
@@ -171,7 +171,7 @@ Inspektor Gadget directly. End-users do not need this and can use
 
 ## Contributing
 
-Contributions are welcome, see [CONTRIBUTING](docs/devel/CONTRIBUTING.md).
+Contributions are welcome, see [CONTRIBUTING](docs/devel/contributing.md).
```

Comment:
I'll point it to the website to avoid too many redirects.

Reply:
mmm, it's actually pointing to the right file, nothing to do here, sorry for the noise.